tutorials/simple-private-token/contracts/private_token_contract/Nargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "private_token"
+type = "contract"
+authors = [""]
+compiler_version = "0.11.1"
+
+[dependencies] 
+aztec = { git="https://github.com/AztecProtocol/aztec-packages/", tag="master", directory="yarn-project/noir-libs/aztec-noir" }
+value_note = { git="https://github.com/AztecProtocol/aztec-packages/", tag="master", directory="yarn-project/noir-libs/value-note"}
```

Comment:
The `aztec-cli compile` command was giving me problems until i added a trailing slash on the first part of the url. Can you confirm this works for you?

Reply:
yeah this was due to a change on the aztec-packages, I have a PR open to fix this on @catmcgee's repo

Reply:
Should we keep this one in the dev rel repo as the source of truth?

Reply:
yes, I'll drop mine 👍 just a quick fix on this one, directory also changed to `yarn-project/aztec-nr/aztec`